src/PerfView/SupportFiles/UsersGuide.htm

```diff
@@ -2437,6 +2437,27 @@ <h2>
                             objects pinned because they are pointed to by a async pinned handle at the time of
                             the heap snapshot.
                         </li>
+                        <li>
+                            <strong><a id="ContentionStacks">Contention Stacks</a></strong> - This view aggregates Contention events.&nbsp;
+                            Contention event is fired when a thread tries to acquire a managed lock that is currently owned
+                            by another thread. Note that each event provides DurationNs event data that
+                            is folded by default. Unfolding it reveals individual pauses of each thread: this can be useful 
+                            to correlate a particular long wait to another events in the trace. Note that not all contention events
+                            are real OS-level waits: the runtime will first spin wait to try acquire the lock fast. The metric
+                            represents the amount of time spent to acquire the lock in milliseconds.
+                        </li>
+                        <li>
+                            <strong><a id="WaitHandleWaitStacks">WaitHandleWait Stacks</a></strong> - This view is similar to <a href="#ContentionStacks">Contention Stacks</a>
```

Comment:
Looks good to me. A couple of suggestions:
- `Contention` events would also be raised by the `Lock` class (`System.Threading.Lock`)
- It may be useful to mention that the `WaitHandleWait` events may not cover all waits, only those that wait on a wait handle (and only alertable ones, which are the typical waits issued from managed code). Waits that bypass the runtime or wait on other synchronous APIs would not raise the event.

Reply:
I rephrased it to include your comments @kouvel , thanks for the review! I didn't know about `System.Threading.Lock`: will it eventually replace `Monitor.Enter` for the usual `lock` construct?

Reply:
Thanks, looks good to me. `Lock` is a new type being introduced, it's separate from `Monitor` and can be used with the `lock` statement. More of the implementation may be shared in the future too, as it is in NativeAOT.

Reply:
Thanks @kouvel!